src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,71 @@
+package lotto.controller;
+
+import lotto.domain.AutoLotto;
+import lotto.domain.WinningLotto;
+import lotto.service.LottoService;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.util.List;
+
+public class LottoController {
+
+    private final InputView inputView;
+    private final LottoService lottoService;
+    private final OutputView outputView;
+
+    public LottoController(InputView inputView, LottoService lottoService, OutputView outputView) {
+        this.inputView = inputView;
+        this.lottoService = lottoService;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        List<AutoLotto> autoLottos = getValidAutoLottoOrRepeat();
+        outputView.lottoCount(autoLottos.stream().count());
+        outputView.lottos(autoLottos);
+        WinningLotto winningLotto = getWinningLottoOrReapeat();
+        setWinningLottoBonusNumberOrRepeat(winningLotto);
+        outputView.winningResult(lottoService.calculateResults(autoLottos, winningLotto));
+        outputView.winningStatisticst(lottoService.calculateWinningStatistics(lottoService.calculateResults(autoLottos, winningLotto), autoLottos));
+    }
```

Comment:
컨트롤러는 사용자의 입력부터 출력까지 핵심 로직이 담겨있는 곳이기 때문에 더욱이 깔끔하게 로직이 보여야 한다고 생각합니다. 
Input과 관련된 DTO생성하여 함수로 분리하면 run 메소드가 더 갈끔해 질 것 같습니다.

그리고 service의 메소드를 호출하는 부분에서 이렇게 하는 것이 더 깔끔하지 않을까요?? 간단하게 수정해보았습니다. 어떻게 생각하시나요?

```suggestion
    public void run() {
        InputDto inputDto = getUserInput(); // *예를 들어서
        
        변수1 = lottoService.calculateResults(autoLottos, winningLotto);
        변수2 = lottoService.calculateWinningStatistics(변수1, autoLottos);
        
        outputView.winningStatisticst(변수2);
    }
```